azavea/pwd-river-dispatches
# River Dispatches

## Overview

Adds initial files, adds `usgs-details.html` to fetch and display the values, sets the main file to use that page with the right `code` parameter for each type, adds basic serving and development scaffolding.

Connects #1
Connects #2
Connects #4 
Connects #5 
Connects #6 
Connects #7 

### Demo

![image](https://user-images.githubusercontent.com/1430060/27233901-222b0e8a-5289-11e7-9020-13b8c6f2298a.png)

### Notes

Need to adjust the styling of the text. We should make it bigger and more centrally located within the circle, but need to be mindful of how the differently long units might wrap.

Also need to check what units show for each type, and maybe adjust them for better presentation.

## Testing Instructions

 * Have the original package unzipped somewhere on your local
 * Clone the repo
 * Run `./scripts/setup /path/to/original/source`
 * Run `./scripts/server`
 * Go to [localhost:9000](http://localhost:9000)
 * Try out all the six options. Make sure you see appropriate values for each type, and _nothing_ for "Sensors"
